Avoids the following error when configuring:

CMake Error in libkvkontakte/CMakeLists.txt:
  Target "kvkontakte" has policy CMP0022 enabled, but also has old-style
  LINK_INTERFACE_LIBRARIES properties populated, but it was exported without
  the EXPORT_LINK_INTERFACE_LIBRARIES to export the old-style properties

which is caused by the project requiring a minimum CMake version that sets
CMP0022 to NEW while the code in KDE4Macros.cmake (in kdelibs) still sets the
deprecated LINK_INTERFACE_LIBRARIES property.
--- CMakeLists.txt
+++ CMakeLists.txt
@@ -56,7 +56,7 @@ add_subdirectory(libkvkontakte)
 # This file contains add_library(bar IMPORTED) statements for each target in the export set, so
 # when loaded later on cmake will create "imported" library targets from these, which can be used
 # in many ways in the same way as a normal library target created via a normal add_library().
-install(EXPORT LibKVkontakteExport DESTINATION ${CMAKECONFIG_INSTALL_DIR} FILE LibKVkontakteTargets.cmake )
+install(EXPORT LibKVkontakteExport DESTINATION ${CMAKECONFIG_INSTALL_DIR} FILE LibKVkontakteTargets.cmake EXPORT_LINK_INTERFACE_LIBRARIES )
 
 # figure out the relative path from the installed Config.cmake file to the install prefix (which may be at
 # runtime different from the chosen CMAKE_INSTALL_PREFIX if under Windows the package was installed anywhere)
